crates/lib/kajiya/src/renderers/rtdgi.rs

```diff
@@ -320,4 +314,18 @@ impl RtdgiRenderer {
 
         filtered_tex.into()
     }
+
+    pub fn create_temporal_filtered_tex(
+        &mut self,
+        rg: &mut rg::TemporalRenderGraph,
+        gbuffer_depth: &GbufferDepth,
+    ) -> rg::Handle<Image> {
+        rg.create(
+            gbuffer_depth
+                .gbuffer
+                .desc()
+                .usage(vk::ImageUsageFlags::empty())
+                .format(vk::Format::R16G16B16A16_SFLOAT),
+        )
+    }
```

Comment:
This can be a 1x1 `R8G8B8A8_UNORM` for the dummy rtdgi texture. Will be cheaper to sample 🙂